docs/extensibility.md

```diff
@@ -29,531 +28,630 @@ pub trait DiscoveryHandler {
+>
+> # Delete Akri CRDs
+> kubectl delete crd/configurations.akri.sh
+> kubectl delete crd/instances.akri.sh
+> ```
+
+Deploy the revised (!) Helm Chart to your cluster:
+
+```bash
+HOST="ghcr.io"
+USER="[[GITHUB-USER]]"
+REPO="${HOST}/${USER}"
+VERS="v$(cat version.txt)-amd64"
+
+sudo microk8s.helm3 install akri ./akri/deployment/helm \
+   --set imagePullSecrets[0].name="${HOST}" \
+   --set agent.image.repository="${REPO}/agent" \
+   --set agent.image.tag="${VERS}" \
+   --set controller.image.repository="${REPO}/controller" \
+   --set controller.image.tag="${VERS}"
 ```
 
-In order to deploy the new, nessie-enabled Akri, we need to build a new Helm chart.  You can follow [these instructions to generate a new Akri chart](./development.md#helm-package). The new Helm chart will be generated in a tgz file called `akri-<VERSION>.tgz` which can be copied to your Kubernetes environment.
+> **NOTE** the Akri SemVer (e.g. `0.0.41`) is reflected in `./version.txt` but the tags must be prefixed with `v` and postfixed with the architecture (e.g. `-amd64`)
+
+Check using `kubectl get pods` and look for a pod named `akri-agent-...` and another named `akri-controller...` and that they're both `RUNNING`.
 
-Assuming you have a Kubernetes cluster running (assuming amd64 for this sample), you can start Akri and apply your Nessie Configuration and watch as broker pods are created.
+Alternatively, you may:
 
-```sh
-# Add secret to give Kubernetes access to your container repo
-kubectl create secret docker-registry <SECRET NAME> --docker-server=ghcr.io  --docker-username=<GITHUB USERNAME> --docker-password=<GITHUB PAT>
-# Use Helm to install your nessie-enabled agent and controller
-helm install akri akri-<VERSION>.tgz \
-    --set imagePullSecrets[0].name="<SECRET NAME>" \
-    --set agent.image.repository="ghcr.io/<GITHUB USERNAME>/agent" \
-    --set agent.image.tag="extensibility-amd64" \
-    --set controller.image.repository="ghcr.io/<GITHUB USERNAME>/controller" \
-    --set controller.image.tag="extensibility-amd64"
-# Apply nessie Akri Configuration
-kubectl apply -f nessie.yaml
-# Watch as agent, controller, and nessie Pods start
-watch kubectl get pods -o wide
+```bash
+kubectl get pods --selector=name=akri-agent
+kubectl get pods --selector=app=akri-controller
 ```
 
+
+## Deploy Broker
+
+Once the HTTP broker has been created, the next question is how to deploy it.  For this, we need the Configuration we created earlier `samples/brokers/http/kubernetes/http.yaml`.  To deploy, use a simple `kubectl` command like this:
+```bash
+kubectl apply --filename=./samples/brokers/http/kubernetes/http.yaml
+```
+
+We can watch as the broker pods get deployed: 
+```bash
+microk8s watch kubectl get pods -o wide
+```
+
+
 ## Contributing your Protocol Implementation back to Akri
 Now that you have a working protocol implementation and broker, we'd love for you to contribute your code to Akri. The following steps will need to be completed to do so:

```

Comment:
I think this can be confusing. People might think they are supposed to contribute HTTP back to Akri. Maybe instead we can say, "While this HTTP protocol may not be useful in real life scenarios, hopefully you now have a grasp on how to extend Akri. If there is a protocol that you think might be useful, you now how the skills to implement it. We'd love for you to contribute your protocol implementation code back to Akri. The following steps..."

Reply:
i have trouble imagining people going through this and thinking that they should contribute our sample back to us.  this is the same wording we had from nessie, so i think for this PR, we should keep it.  if we want to change it, lets do that in another PR.